src/__test__/navbar.test.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import NavBar from "../components/navbar";
+import "@testing-library/jest-dom";
+import { useLocation } from "react-router-dom";
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useLocation: jest.fn(),
+}));
+
+describe("NavBar component", () => {
+  let mockUseLocation;
+
+  beforeEach(() => {
+    mockUseLocation = useLocation.mockImplementation(() => ({
+      pathname: "/some/path",
+    }));
+  });
+
+  afterEach(() => {
+    mockUseLocation.mockRestore();
+  });
+
+  test("renders without errors", () => {
+    render(
+      <BrowserRouter>
+        <NavBar />
+      </BrowserRouter>
+    );
```

Comment:
- [ ] Good work so far, however as mentioned by the previous reviewer https://github.com/jeanbuhendwa/covidApp/pull/6#discussion_r1171229283 please add more test cases, as required in the project too :point_down: 
![image](https://user-images.githubusercontent.com/97521557/233180017-26d524e1-a61c-4fef-9da3-9a3d49e9de9a.png)

currently there is only test case for the navbar component, please add test case for the homepage and detailspage component, and you can also add tests for checking the reducer by providing a predefined state and action and expecting the reducer to behave accordingly,